REQUIRE

```diff
@@ -3,4 +3,6 @@ CategoricalArrays
 Tables
 StatsBase 0.22.0
 DataStructures
-Missings
\ No newline at end of file
+Missings
+SparseArrays
+LinearAlgebra
```

Comment:
Project.toml needs all dependencies including stdlib
REQUIRE breaks if you include stdlib

Reply:
:sweat_smile: